examples/tls.mts

```diff
@@ -0,0 +1,41 @@
+import {Cluster, VerifyMode} from "../index.js"
+
+const nodes = process.env.CLUSTER_NODES?.split(",") ?? ["localhost:9142"];
+console.log(`Connecting to ${nodes}`);
+
+const cluster = new Cluster({
+    nodes,
+    ssl: {
+        enabled: true,
+        truststoreFilepath: "/your/path/to/certificates/client_cert.pem",
+        privateKeyFilepath: "/your/path/to/certificates/client_key.pem",
+        caFilepath: "/your/path/to/certificates/client_truststore.pem",
```

Comment:
I will have to follow your article to see how can I actually integrate this, as this examples requires another files I think I will change the structure to `examples/tls/index.mts` and add the .pem files there so the example become fully functional in the perspective of an user being able to clone the repo and run this example without having to change anything.